test/integration/narrow/routes/capture.test.js

```diff
@@ -0,0 +1,81 @@
+describe('Capture test', () => {
+  jest.mock('ffc-messaging')
```

Comment:
Tidy up of mocks/imports

Have all mocks defined together, then all requires, then local const/let variables -- use whitespace to break up

Makes it easier for me to what's being brought for this tests dependencies

Same for all routes changed

Reply:
Tidied up :)